### DISCUSSION TITLE
What is Avalonia.Fonts.Inter truly using for?

### DISCUSSION THREAD

**Discussion by kyuranger** [AvaloniaUI/Avalonia#11413] [Q&A] [answered]
In newest Avalonia template (I am not sure whether is Preview 7 or Preview 8), it will add a new nuget package 

> Avalonia.Fonts.Inter

And also, in Program.cs will add a new line code 

> WithInterFont

:
```
 public static AppBuilder BuildAvaloniaApp()
            => AppBuilder.Configure<App>()
                .UsePlatformDetect()
            .WithInterFont()
                .LogToTrace();
```

What will the project benefit from this new package? Thank you.

**Comment by maxkatz6** [comment, marked as answer]
It's a custom font - Inter. Open source alternative to Segoe font. 

Without that app would use system font which might be different on different OS.

**Reply by maxkatz6** [reply]
Previously in Fluent theme this font was always included. Which was unnecessary increasing binary sizes for apps that don't need inter.

**Reply by kyuranger** [reply]
> Previously in Fluent theme this font was always included. Which was unnecessary increasing binary sizes for apps that don't need inter.

Got it. Thank you.
